build/Build.Pack.cs

```diff
@@ -30,9 +31,9 @@ partial class Build
     [Parameter("Specifies the platforms to build the docker images in. Multiple platforms must be comma-separated. Defaults to 'linux/arm64,linux/amd64'.",
         Name = "DockerPlatform")]
     string DockerPlatform = "linux/arm64,linux/amd64";
-    
-    [PathExecutable]
-    readonly Tool Multipass;
+
+    [PathVariable]
```

Comment:
Why this change?

Reply:
Changes in the Nuke Attribute. `PathVariableAttribute` replaces the obsolete `PathExecutableAttribute`

https://github.com/nuke-build/nuke/blob/6ed9e4d9c7476444eb6b984591d1364b47723144/source/Nuke.Common/Attributes/PathVariableAttribute.cs#L55-L56